extension/src/generate.ts

```diff
@@ -3,7 +3,13 @@ import vscode from "vscode";
 import * as ailly from "@ailly/core";
 import { FileSystem } from "@davidsouther/jiffies/lib/esm/fs.js";
 import { VSCodeFileSystemAdapter } from "./fs.js";
-import { LOGGER, getAillyEngine, getAillyModel, resetLogger } from "./settings";
+import {
+  LOGGER,
+  getAillyAwsProfile,
+  getAillyEngine,
+  getAillyModel,
+  resetLogger,
```

Comment:
Unused imports?

Reply:
Oh, wait, nm.

Reply:
No, just got reformatted because it grew too long.